extensions/arc/src/extension.ts

```diff
@@ -54,6 +50,19 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
 	});
 
 	await checkArcDeploymentExtension();
+
+	return {
+		getAzureArcTreeDataProvider: () => treeDataProvider
+	};
+}
+
+async function connectToController(treeDataProvider: AzureArcTreeDataProvider) {
```

Comment:
Why did you move this out? Just leave it in the command definition unless something else needs to use this.

Reply:
I was going to create a connectToController method as an API method but decided to postpone that to later. So I will revert this change for now.

Reply:
reverted